app/models/riiif/file.rb

```diff
@@ -57,8 +57,8 @@ def extract_tifftopnm(options)
         command << "pnmscalefixed -ysize #{h}"       if h.present? && w.blank?
       end
 
-      # ignore quality
-      command << "pnmtojpeg -quality 95"
+      # ignoring options[:quality] for jpeg here
```

Comment:
Technically the "quality" the original comment refers to is the IIIF Image API "quality" (https://iiif.io/api/image/2.1/#quality) which has nothing to do with JPEG quality. We're only generate the "default" quality, not letting the URL specify color/grayscale/etc.